WindowsFormsApp1/Form1.cs

```diff
@@ -15,15 +15,16 @@ namespace WindowsFormsApp1
 
     public partial class Form1 : Form
     {
-
-        //RequestCurrency currency = new RequestCurrency();
-
-        Calculator calc = new Calculator();
+        string partialExpression;
+        IEnumerable<IOperator> operations = new IOperator[] { new AdditionOperator(), new SubtractionOperator(), new MultiplicationOperator(), new DivisionOperator() };
+        Calculator calc;

```

Comment:
to truly have this coded against interfaces this must be `ICalculator` otherwise you aren't using the interface, just the class.